### PR TITLE
[Graphview]Fix tag colors on boxes[gramps51]

### PR DESCRIPTION
**Pull request by romjerome**

### CODE REVIEW

**Comment by romjerome** [comment]
fix [10721](https://gramps-project.org/bugs/view.php?id=10721)

**Comment by romjerome** [comment]
#86

File: GraphView/graphview.py [hunk 1]
```diff
@@ -2377,6 +2377,12 @@ def get_tags_and_table(self, obj):
             tag_table = ('<TABLE BORDER="0" CELLBORDER="0" '
                          'CELLPADDING="5"><TR>')
             for tag in tags:
+                rgba = Gdk.RGBA()
+                rgba.parse(tag.get_color())
+                value = '#%02x%02x%02x' % (int(rgba.red * 255),
+                                   int(rgba.green * 255),
+                                   int(rgba.blue * 255))
+                tag.set_color(value)
```
Comment (vantu5z) [inline GraphView/graphview.py]: Should we really need to set color to tag?
`tag.set_color(value)`
Maybe just use generated value?
`tag_table += '<TD BGCOLOR="%s"></TD>' % value`

Comment (romjerome) [reply GraphView/graphview.py]: I do not know ... Does it work too?
In my mind, it was for re-assigning
a value after the conversion, 
like done by the hand via GUI.

Yes, feel free. Thanks.

Comment (romjerome) [reply GraphView/graphview.py]: Note, as there is no DB commit,
I only supposed that [setting color](https://www.gramps-project.org/docs/gen/gen_lib.html?highlight=tag#gramps.gen.lib.tag.Tag.set_co) 
was in memory.

Comment (vantu5z) [reply GraphView/graphview.py]: Ok for me as we don't commiting to db.

**Comment by sam-m888** [comment]
Mentioned on bug reported as already having been mentioned here

https://github.com/gramps-project/addons-source/pull/86#issuecomment-327999074

**Comment by sam-m888** [comment]
@romjerome Can you please add a screenshot of before and after , so I can work out what I should be looking ?

**Comment by romjerome** [comment]
@sam-m888 I can make it better ...
I will use your screenshot! 
img_20170908_130447727

See your [comment](https://github.com/gramps-project/addons-source/pull/86#issuecomment-327987719) on pull request #86

**Comment by sam-m888** [comment]
Thank you that helps, so the issue is that the Chart was displaying the wrong colors compared to the tooltip & "Organise tags" dialog 👍   Tested and confirmed.

Before:
2019-09-29 11_57_28-Window

After: 

2019-09-29 11_59_07-Window

2019-09-29 11_58_53-Window

**Comment by sam-m888** [comment]
Thank you

**Pull request merged**